syne_tune/optimizer/schedulers/searchers/bore/bore.py

```diff
@@ -246,7 +244,8 @@ def wrapper(x):
                 f"config={config}] "
                 f"optimization time : {opt_time}"
             )
-            self._excl_list.add(config)  # Should not be suggested again
+            if not self.allow_duplicates:
+                self._excl_list.add(config)  # Should not be suggested again
```

Comment:
can't we delegate to parent class as well?

Reply:
I'll  try something